docs/topics/exporters.rst

```diff
@@ -216,6 +216,11 @@ BaseItemExporter
       encoding). Other value types are passed unchanged to the specific
       serialization library.
 
+   .. attribute:: indent_width
+
+      Amount of spaces used to indent the output on each level.
+      Defaults to ``None``, which disables indentation.
```

Comment:
This argument controls newlines, not only indent spaces. I wonder if we should follow the same approach as stdlib json:

> If indent is a non-negative integer, then JSON array elements and object members will be pretty-printed with that indent level. An indent level of 0, or negative, will only insert newlines. None (the default) selects the most compact representation.